migrations/2022_05_14_180552_create_notifications_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('notifications', function (Blueprint $table) {
+            $table->id();
```

Comment:
the notification should have:
- text
- user_id (and a db relation)
- status( if your db supports enum use it) which could be NEW:0, SENT:1, READ:2 or you could even use separate fields per each state(boolean)

Reply:
done

Reply:
you need first to declare the user_id
then to index it and then to add the foreign key eg.
```
Schema::create('pets', function (Blueprint $table) {
            $table->bigIncrements('id');
            $table->string('name', 40)->nullable(true);
            $table->string('type',10)->nullable(false);
            $table->string('breed', 100)->nullable(false);
            $table->string('chip', 100)->nullable(true);
            $table->date('birthday')->nullable(true);
            $table->smallInteger('weight')->nullable(true);
            $table->unsignedBigInteger("client_id")->nullable(false);
            $table->boolean('alive')->default(true);
            $table->timestamps();
            $table->softDeletes();

            $table->index("client_id");
            $table->foreign("client_id")
                ->references('id')->on('clients')
                ->onDelete('cascade');
        });
```

Reply:
we do the same for all columns that are relations. so for comment id as well